core/client/components/gh-trim-focus-input.js

```diff
@@ -5,16 +5,20 @@ var TrimFocusInput = Ember.TextField.extend({
     attributeBindings: ['autofocus'],
 
     autofocus: Ember.computed(function () {
```

Comment:
Nah, my thinking didn't go much beyond "Hey, there's a `get` without a
dependency declaration"
Perhaps there could be a very small chance of a race condition if whatever
property focus was set to changed between the time the input was created
and when it was rendered, but meh. That's me just reaching for a reason
other than code convention

On Sat, Jan 31, 2015 at 12:49 PM, Jason Williams <notifications@github.com>
wrote:

> In core/client/components/gh-trim-focus-input.js
> <https://github.com/TryGhost/Ghost/pull/4872#discussion_r23891497>:
>
> > @@ -5,16 +5,20 @@ var TrimFocusInput = Ember.TextField.extend({
> >      attributeBindings: ['autofocus'],
> >
> >      autofocus: Ember.computed(function () {
>
> I'm thinking no because autofocus seems like it should be determined once
> when the component is rendered and that's it.
>
> Are you thinking of a case where it would be useful? (to be honest, I'm
> not even sure if adding the autofocus attribute after the fact would
> trigger the browser to do anything...)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TryGhost/Ghost/pull/4872/files#r23891497>.
>